pdsClient/api.go

```diff
@@ -64,18 +64,31 @@ type AllowedRead struct {
 	ContentType string `json:"content_type"`
 }
 
-// InternalModifiedSearchRequest represents a valid request to internal records search for getting modified records
-type InternalModifiedSearchRequest struct {
-	NextToken int                   `json:"next_token"`
-	Range     InternalModifiedRange `json:"range"`
+// InternalSearchRequest represents a valid request to the internal records search endpoint
+type InternalSearchRequest struct {
+	NextToken    int                    `json:"next_token,omitempty"`
+	Range        *InternalModifiedRange `json:"range,omitempty"`
```

Comment:
Range can also be empty right? If I wanted to just search on writerIDs/UserID/ContentType would it also work?

Reply:
Correct